nimbus-ui/nimbusui/src/app/components/platform/validators/custom.validators.ts

```diff
@@ -74,5 +74,13 @@ export class CustomValidators {
         };
     }
 
+    static past(control: FormControl) {
+        return null;
```

Comment:
I'm not sure if returning null here is the best option.  @sandeepmantha thoughts?

Reply:
Does not look quite right. What is the intension of this ?

Reply:
@sandeepmantha  i have  modified and committed the code. Can you please take a look at latest changes in this PR. The intention is not to perform any validation here. I just need whether the param is annotated with @Past or @Future.